src/components/ReportActionItemIOUQuote.js

```diff
@@ -1,29 +1,50 @@
 import React from 'react';
-import {View} from 'react-native';
+import {View, Text} from 'react-native';
 import PropTypes from 'prop-types';
 import _ from 'underscore';
-import styles from '../styles/styles';
+import styles, {webViewStyles} from '../styles/styles';
 import ReportActionPropTypes from '../pages/home/report/ReportActionPropTypes';
-import RenderHTML from './RenderHTML';
 
 const propTypes = {
     // All the data of the action
     action: PropTypes.shape(ReportActionPropTypes).isRequired,
+
+    // Should the View Details link be displayed?
+    showViewDetailsLink: PropTypes.bool,
+
+    // Callback invoked when View Details is pressed
+    onViewDetailsPressed: PropTypes.func,
+};
+
+const defaultProps = {
+    showViewDetailsLink: false,
+    onViewDetailsPressed: null,
 };
 
-const ReportActionItemIOUQuote = ({action}) => (
+const ReportActionItemIOUQuote = ({action, showViewDetailsLink, onViewDetailsPressed}) => (
     <View style={[styles.chatItemMessage]}>
-        {_.map(action.message, (fragment, index) => {
-            const viewDetails = '<br /><a href="#">View Details</a>';
-            const html = `<blockquote>${fragment.text}${viewDetails}</blockquote>`;
-            return (
-                <RenderHTML key={`iouQuote-${action.sequenceNumber}-${index}`} html={html} />
-            );
-        })}
+        {_.map(action.message, (fragment, index) => (
+            <View key={`iouQuote-${action.sequenceNumber}-${index}`}>
+                <View style={[webViewStyles.tagStyles.blockquote]}>
```

Comment:
NAB, slightly unexpected to see the `webViewStyles` being reused for something other than the `web view` (which I think is the render html stuff). Probably we should generalize this style rather than reuse it with a name that implies it's meant for something else.